src/Rings/mpoly-localizations.jl

```diff
@@ -1475,13 +1475,31 @@ end
 function intersect(I::MPolyLocalizedIdeal, J::MPolyLocalizedIdeal)
   L = base_ring(I)
   L == base_ring(J) || error("ideals must be defined in the same ring")
-  preI = Oscar.pre_saturated_ideal(I)
-  preJ = Oscar.pre_saturated_ideal(J) 
-  R = base_ring(L)
+  preI = pre_saturated_ideal(I)
+  preJ = pre_saturated_ideal(J)
   K = intersect(I, J)
   return L(K)
 end
 
+function intersect(I::MPolyLocalizedIdeal,J::MPolyLocalizedIdeal...) 
+  L = base_ring(I)
+  erg = pre_saturated_ideal(I)
+  for K in J
+    base_ring(K) == L || error("base rings must match")
+    erg = intersect(erg,pre_saturated_ideal(K))
+  end
+  return L(erg)
+end
+
+function intersect(VI::Vector{<:MPolyLocalizedIdeal{T}}) where T
```

Comment:
So far none of the `intersect` functions handle this case. They all require `length(...) > 0`. It might be more similar to the `prod()/sum()` functions? This has an `init` keyword argument to handle the empty case.

I suggest we just ignore it for now.

Reply:
Thank you for the quick reply.